### PR TITLE
Sale partner primeship add security access

### PR DESCRIPTION
**Pull request by Kev-Roche**
add model access to partner_primeship
cc @hparfr @paradoxxxzero

### CODE REVIEW

**Comment by hparfr** [comment]
@Kev-Roche may you add a little description about the proposed module in order to ease the review

**Comment by hparfr** [comment]
@paradoxxxzero do you agree to change license to AGPL-3 (it's the license of the dependency)?

**Comment by paradoxxxzero** [comment]
No problem ¯⁠\⁠_⁠(⁠ツ⁠)⁠_⁠/⁠¯

**Comment by nayatec** [comment]
amend model access
cc @hparfr @paradoxxxzero

File: sale_partner_primeship/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+    primeship_id = fields.Many2one(
+        string="Primeships",
+        comodel_name="sale.primeship",
+        compute="_compute_primeship_id",
+        inverse="_inverse_primeship_id",
+    )
+
+    # One2one impl
+    primeship_ids = fields.One2many(
```
Comment (hparfr) [inline sale_partner_primeship/models/sale_order.py:61]: does it need an sql constraint ?

File: sale_partner_primeship/models/sale_order.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        comodel_name="sale.primeship", inverse_name="order_line_id"
+    )
+
+    @api.depends("primeship_ids")
+    def _compute_primeship_id(self):
+        for record in self:
+            if record.primeship_ids:
+                record.primeship_id = record.primeship_ids[0]
```
Comment (hparfr) [inline sale_partner_primeship/models/sale_order.py:69]:
```suggestion
            record.primeship_id = record.primeship_ids[:1]
```

File: sale_partner_primeship/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,3 @@
+This module adds a concept of primeship for partners.
+This primeship is activated on the confirmation of a sale containing
+a primeship activation product for a customizable duration.
```
Comment (hparfr) [inline sale_partner_primeship/readme/DESCRIPTION.rst:3]: describe what is a premiship

specify only one active primeship at the time and only one time of "type" primeship

**Review by hparfr — COMMENTED** [review]
please add yoursevles ass maintainers  and add some tests

**Comment by rousseldenis** [comment]
@Kev-Roche What's the status of this?

**Comment by nayatec** [comment]
@Kev-Roche @rousseldenis has been superseded by #2815 but remarks from @hparfr here still needs to be adressed

**Comment by rousseldenis** [comment]
Closing in favor of #2815

**Pull request closed without merging**